src/pkg/utils/auth.go

```diff
@@ -60,7 +63,6 @@ func credentialParser(path string) ([]Credential, error) {
 	if err != nil {
 		return nil, err
 	}
-	defer file.Close()
```

Comment:
I believe we still want this to be a defer

Reply:
Mentioned this below, but, while this should get handled linearly without getting ejected from an error there's no harm in guarding against panics w/ a defer